source/glimmer-components/handling-user-interaction.md

```diff
@@ -0,0 +1,121 @@
+Components allow you to define UI controls that can be reused throughout
+your application. If they're generic enough, they can even be shared across
+multiple applications.
+
+In order to make a useful control, we would need to allow our users to
+interact with our Components and handle these interaction somehow. Let's
```

Comment:
This may be nit-picky too early, but seems like `Component` is capitalized throughout these docs even when not at the beginning of a sentence.  That seems different from the way we use it in the current docs ...